[2024-10-02T18:59:16.580+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:59:16.602+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-14T00:00:00+00:00 [queued]>
[2024-10-02T18:59:16.615+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-14T00:00:00+00:00 [queued]>
[2024-10-02T18:59:16.616+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:59:16.837+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-14 00:00:00+00:00
[2024-10-02T18:59:16.846+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6230) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:59:16.848+0000] {standard_task_runner.py:63} INFO - Started process 6246 to run task
[2024-10-02T18:59:16.847+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-14T00:00:00+00:00', '--job-id', '1288', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp8yzsyioe']
[2024-10-02T18:59:16.850+0000] {standard_task_runner.py:91} INFO - Job 1288: Subtask carregar
[2024-10-02T18:59:16.899+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-14T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:59:17.008+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-14T00:00:00+00:00'
[2024-10-02T18:59:17.009+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:59:17.071+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:59:17.072+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:59:17.083+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-14T00:00:00+00:00, execution_date=20240114T000000, start_date=20241002T185916, end_date=20241002T185917
[2024-10-02T18:59:17.142+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:59:17.162+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:59:17.165+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:25:33.082+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:25:33.123+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-14T00:00:00+00:00 [queued]>
[2024-10-02T19:25:33.403+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-14T00:00:00+00:00 [queued]>
[2024-10-02T19:25:33.404+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:25:33.432+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-14 00:00:00+00:00
[2024-10-02T19:25:33.442+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9578) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:25:33.444+0000] {standard_task_runner.py:63} INFO - Started process 9599 to run task
[2024-10-02T19:25:33.445+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-14T00:00:00+00:00', '--job-id', '2293', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp27fwy3nc']
[2024-10-02T19:25:33.448+0000] {standard_task_runner.py:91} INFO - Job 2293: Subtask carregar
[2024-10-02T19:25:33.521+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-14T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:25:33.678+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-14T00:00:00+00:00'
[2024-10-02T19:25:33.698+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:25:33.916+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:25:33.922+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:25:33.943+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-14T00:00:00+00:00, execution_date=20240114T000000, start_date=20241002T192533, end_date=20241002T192533
[2024-10-02T19:25:34.021+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:25:34.040+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:25:34.042+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
